### PR TITLE
Some Persona Cleanups 

### PR DESCRIPTION
**Pull request by lisaqyli**
### Description

This is a PR to clean up Persona, here are some notable changes:
1. should pass user's MTW address to firebase
2. UIs for different Persona Inquiry states
3. on inquiry Cancelled flow, kycStatus will be updated to `Pending`, updated Pending UI to let user to start over.

### Tested
Manually tested

### CODE REVIEW

File: packages/mobile/src/account/LinkBankAccountScreen.tsx [hunk 1]
```diff
@@ -98,10 +128,33 @@ function StepOne({ kycStatus }: StepOneProps) {
           <Text style={styles.description}>{t('linkBankAccountScreen.pending.description')}</Text>
         </View>
       )
-    case KycStatus.NotCreated:
-    case KycStatus.Created:
```
Comment (cajubelt) [inline packages/mobile/src/account/LinkBankAccountScreen.tsx:102]: So what will happen now for the notcreated and created cases? Maybe I'm missing something.

**Review by cajubelt — APPROVED** [review]
Thanks for cleaning things up! 🚢 

Left a question, probably fine though so I'm also approving.

In the future, please try for single-purpose PR's.

File: packages/mobile/src/account/LinkBankAccountScreen.tsx (hunk 1 above)
Comment (jh2oman) [reply packages/mobile/src/account/LinkBankAccountScreen.tsx:102]: The not created and created pieces get the 'default' from the case/switch, which is the "Verify your Identity" view

File: packages/mobile/src/account/LinkBankAccountScreen.tsx [hunk 2]
```diff
@@ -98,10 +128,33 @@ function StepOne({ kycStatus }: StepOneProps) {
...
+    case KycStatus.Completed:
```
Comment (jh2oman) [inline packages/mobile/src/account/LinkBankAccountScreen.tsx:131]: I thought that "Completed" meant that the user completed the KYC but it was not yet approved, which was why we previously showed the "Verification Pending" screen. Why are we now showing the "Verification Complete" screen?

Comment (lisaqyli) [reply packages/mobile/src/account/LinkBankAccountScreen.tsx:131]: you can see the description of various persona lifecycle states here: https://docs.withpersona.com/docs/models-lifecycle
And also i tested this, if the inquiry was successful without needing to be manually reviewed, `completed` will be the updated state

**Comment by ValoraQA** [comment]
Hey @MuckT, Please let us know do we need to perform testing from our side.

**Pull request merged**